src/app/components/surveillance/complaints/complaints.jsx

```diff
@@ -69,7 +70,7 @@ function ChplComplaints(props) {
   /* eslint-disable react/destructuring-assignment */
   const [complaints, setComplaints] = useState([]);
   const [page, setPage] = React.useState(0);
-  const [rowsPerPage, setRowsPerPage] = React.useState(5);
+  const [rowsPerPage, setRowsPerPage] = React.useState(50);
```

Comment:
What should happen with the options that are available in the dropdown? Should the "5" still be an option? What about the "25"?

Reply:
Our original tables go by 50 | 100 | 200 was this a decision made in the past?

Reply:
I doubt there was any particular reason chosen for those numbers; they were probably just picked randomly and then never thought about again

Reply:
Sorry I didn't see this comment last week. However, there's no common rule out there for pagination length, that indicates we should have 5 | 10 | 20 or 50 | 100 | 200.. Its depends on the data/product being shown. My thought is since CHPL is a service app and not a eCommerce site we should use the larger number set. This will be our common rule for table pagination moving forward 50 | 100 | 200. We can make a decision later down the road on specific page/table, if needed.